src/main/java/org/myrobotlab/math/MapperLinear.java

```diff
@@ -18,13 +18,11 @@
   Double minY;
   Double maxY;
 
-  // clipping
-  Double minOutput;
-  Double maxOutput;
+  // limits

```

Comment:
Doesn't dropping the word "output" from these variables make it more confusing?  I feel like we should have 4 parameters for the mapper.   inputMin, inputMax, outputMin, outputMax ...  There have been many times when people were confused about input vs output max/min settings for mappings..

Reply:
After thinking about it for a while, there is no need for input limits and output limits - If the point is to calculate output, output limits are only needed.  

![mapper1](https://user-images.githubusercontent.com/1266144/44403189-a6b4fa80-a508-11e8-86d8-61ebd539b99b.png)
This is a minimal mapper

I think there was way more confusion about the composite of mappers F(G(x)) - which the First Law of MotorControl has removed (Yay!)
![mapper2](https://user-images.githubusercontent.com/1266144/44403370-24790600-a509-11e8-9cc2-2e0804730ef8.png)

This is the composite, and it doesn't work and I would say is responsible for 90% of the confusion - its immensely more simple if some default values are put into the MotorControl mapper and it is used directly.